src/components/MultiStepIndicator/MultiStepIndicator.module.scss

```diff
@@ -9,4 +9,10 @@
   margin: 16px;
   height: 1px;
   width: 48px;
+
+  &.compact {
+    width: 16px;
+    margin-left: var(--spacing-small);
```

Comment:
let's switch to margin-inline-start and margin-inline-end

Reply:
Today, it is considered a best practice in the industry to utilize CSS that remains unaffected by the orientation of text (i.e., right-to-left or left-to-right). This approach ensures easier support for right-to-left languages in the future.

Reply:
Nice, thanks